### PR TITLE
[jsk_robot_startup/smach_to_mail] Set sender address and receiver address from rosparam

### PR DESCRIPTION
**Pull request by tkmtnt7000**
This PR enables to set mail address from rosparam.
Related to https://github.com/jsk-ros-pkg/jsk_robot/pull/1558

### CODE REVIEW

File: jsk_robot_common/jsk_robot_startup/scripts/smach_to_mail.py [hunk 1]
```diff
@@ -29,8 +29,8 @@ def __init__(self):
         self.bridge = CvBridge()
         self.smach_state_list = {}  # for status list
         self.smach_state_subject = {}  # for status subject
-        self.sender_address = "tsukamoto@jsk.imi.i.u-tokyo.ac.jp"
-        self.receiver_address = "tsukamoto@jsk.imi.i.u-tokyo.ac.jp"
+        self.sender_address = rospy.get_param("~sender_address")
```
Comment (k-okada) [inline jsk_robot_common/jsk_robot_startup/scripts/smach_to_mail.py:32]: @tkmtnt7000 thank you, please add error checking code, when no param is set, use default value or raise errors

**Review by k-okada — CHANGES_REQUESTED** [review]

File: jsk_robot_common/jsk_robot_startup/scripts/smach_to_mail.py (hunk 1 above)
Comment (tkmtnt7000) [reply jsk_robot_common/jsk_robot_startup/scripts/smach_to_mail.py]: Sorry for late.
I add error checking code(https://github.com/k-okada/jsk_robot/pull/71/commits/354bd5a58d9acbadb820a140f35e4cecae8b5aca)

**Comment by tkmtnt7000** [comment]
I think there might be some demand that we don't want to write their email address (robot or personal one) on github, so I make it possible to set the email address from the yaml file as well. The implementation is based on email_topic.py.

**Comment by k-okada** [comment]
@tkmtnt7000 pleaes update documents,
https://github.com/jsk-ros-pkg/jsk_robot/tree/master/doc/jsk_robot_common/jsk_robot_startup

> I think there might be some demand that we don't want to write their email address (robot or personal one) on github, so I make it possible to set the email address from the yaml file as well. The implementation is based on email_topic.py.

In my opinion, we need to be very careful on using `/var/lib`, 
1) we put information on '/var/lib/robot' because we do not want to share these inforamtion on CVS
2) Most of email address is already listed on package.xml. 
3) The more information we put in '/var/lib/robot', the more  we want to put these information on CVS
```
T:今後この辺りもバージョン管理できたら履歴が残っていいかも、と思いました
Y:/var/lib以下ってこと？この辺りというのは
T:あ、そうです
```
4) may be I sould not allow to use yaml file in https://github.com/jsk-ros-pkg/jsk_robot/pull/1388 ...
5) If we want to use system-wide information, we should use rosparam, so that you can access from different machine.

--> Please discuss with @708yamaguchi and others, if we can remove `/var/lib/robot/email_topic.yaml`, please remove https://github.com/k-okada/jsk_robot/pull/71/commits/e58ac0775e1c8e5db86ceaf2ae33c28795745098, if that features is useful or already widely used, please simply update the document.

**Comment by tkmtnt7000** [comment]
> @tkmtnt7000 pleaes update documents,
> [https://github.com/jsk-ros-pkg/jsk_robot/tree/master/doc/jsk_robot_common/jsk_robot_startup](https://github.com/jsk-ros-pkg/jsk_robot/tree/master/doc/jsk_robot_common/jsk_robot_startup?rgh-link-date=2022-08-23T02%3A43%3A12Z)

OK. I'll write document about smach_to_mail.py overview first.

> In my opinion, we need to be very careful on using /var/lib,
> 
> we put information on '/var/lib/robot' because we do not want to share these inforamtion on CVS
> Most of email address is already listed on package.xml.
> The more information we put in '/var/lib/robot', the more we want to put these information on CVS
> T:今後この辺りもバージョン管理できたら履歴が残っていいかも、と思いました
> Y:/var/lib以下ってこと？この辺りというのは
> T:あ、そうです
> may be I sould not allow to use yaml file in https://github.com/jsk-ros-pkg/jsk_robot/pull/1388 ...
> If we want to use system-wide information, we should use rosparam, so that you can access from different machine.
> --> Please discuss with @708yamaguchi and others, if we can remove /var/lib/robot/email_topic.yaml, please remove https://github.com/k-okada/jsk_robot/commit/e58ac0775e1c8e5db86ceaf2ae33c28795745098, if that features is useful or already widely used, please simply update the document.

I understand. I will discuss with @708yamaguchi and other members whether to use `/var/lib` or not.
Surely, I also think that the more parts of the system that are not shared on CVS, the harder it is to find problems when they occur....

**Pull request merged**